jwt-core/src/main/scala/tsec/jwt/JWTClaims.scala

```diff
@@ -269,6 +269,16 @@ object JWTClaims extends JWSSerializer[JWTClaims] {
       }
   }
 
+  // Default Decoder[Either[A, B]] is not used (see: https://tools.ietf.org/html/rfc7519#section-4.1.3)
+  // Also this custom decoder "not" polymorphic as its behaviour could be inconsistent for other A and B
+  implicit val audienceDecoder: Decoder[Either[String, List[String]]] = {
+    c: HCursor => 
+      c.as[String] match {
+        case Right(a) => Right(Left(a))
```

Comment:
This sort of making me not very sold on the signature. Maybe an `ADT` which you match on would be nice, but then we wouldn't get all of the stdlib goodies from `Either`

Reply:
An ADT seems apropriate here, I'm not sure how many people use the methods on Etiher from stdlib now but we could provide a `toList` method on the trait since in most case operations on the singleton list (e.g. finding the services own url in the list) will be equivalent to what would be done on the string literal (e.g. comparison).